src/components/root_layout.tsx

```diff
@@ -17,12 +20,46 @@ interface RootLayoutProps {
  * Provides a banner that includes a menu
  */
 const RootLayout: FC<RootLayoutProps> = ({children}) => {
+
+  const location = useLocation();
+
+  // Regular expression pattern to match the desired paths
+  const regexPatternToHome = /^(.*)\/app\/([^\/]+)$/;
+  const regexPatternToHPWH = /^(.*)\/app\/qa_hpwh\/([^\/]+)$/
+  const regexPatternToPlayGround = /^(.*)\/app\/playground\/([^\/]+)$/
+
+  // Determine whether to show the back button based on the current route
+  const showBackButtonToHome = regexPatternToHome.test(location.pathname);
+  const showBackButtonToHPWH = regexPatternToHPWH.test(location.pathname);
+  const showBackButtonToPlayGround = regexPatternToPlayGround.test(location.pathname);
+  
   return (
     <div style={{marginLeft: "auto", marginRight: "auto", maxWidth: 800, backgroundColor: "rgba(231, 231, 231)"}}>
       {/* TODO: Add a menu */}
       <NavBar style={{backgroundColor: "green"}}>
-        <Container>
-          <NavBar.Brand><span style={{color: "gold", fontSize: "2rem"}}>Quality Install Tool</span></NavBar.Brand>
+        {showBackButtonToHome && (
+          <Link to="/" style={{ textDecoration: "none" }}>
```

Comment:
We should change the route to the home view to be "/app".
In the router, "/" should also reroute to "/app".